src/main/kotlin/thatline/localup/chatgpt/service/ChatGptService.kt

```
@@ -41,26 +52,200 @@ class ChatGptService(
+            - 각 섹션마다 빈 줄을 추가하여 단락을 명확히 구분하세요.
+            - 긴 문장은 적절한 위치에서 줄바꿈하여 읽기 쉽게 만드세요.
+            - 번호 목록: 1. 2. 3. 형식 사용
+            - 하위 항목: - 또는 · 기호 사용
+            - 제목이나 강조가 필요한 부분은 "대괄호 [강조내용]" 형식 사용
+
+            특히 현재 대화하는 사용자는 다음과 같은 사업정보를 가진 사업자입니다:
+            - 업체명: ${business.name}
+            - 업종: ${business.type} (${business.item})
+            - 위치: ${business.address}${if (!business.addressDetail.isNullOrBlank()) " ${business.addressDetail}" else ""}
+            - 좌석 수: ${business.seatCount}석
+            - 평균 객단가: ${String.format("%.0f", business.averageOrderAmount)}원
```

detekt (1.23.8)

[warning] String.format("%.0f", business.averageOrderAmount) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)